db/schema_0.sql

```diff
@@ -657,6 +657,17 @@ CREATE TABLE `twilio_delivery_status` (
   CONSTRAINT `twilio_delivery_status_message_id_ibfk` FOREIGN KEY (`message_id`) REFERENCES `message` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+
+DROP TABLE IF EXISTS `twilio_retry`;
+CREATE TABLE `twilio_retry` (
+  `message_id` bigint(20) NOT NULL,
+  `retry_id` bigint(20) NOT NULL,
+  PRIMARY KEY (`message_id`),
```

Comment:
I'm wondering if you could make this use a composite primary key of the two columns, instead of PK'ing message_id and creating a separate index for retry_id

Reply:
I think two unique indexes are the constraints we want. Each message can have exactly one retry entry (enforced by message_id PK), and no message can be a retry for more than one message (unique retry_id). Composite key would let us have multiple retries, which might make sense in the future, but keeping unique retry_id is still a good thing to have.

Reply:
That makes sense